src/js/serial_backend.js

```diff
@@ -676,33 +674,35 @@ function update_live_status() {
         MSP.send_message(MSPCodes.MSP_ANALOG, false, false);
     }
 
-    var active = ((Date.now() - FC.ANALOG.last_received_timestamp) < 300);
+    const active = ((Date.now() - FC.ANALOG.last_received_timestamp) < 300);
 
-    for (var i = 0; i < FC.AUX_CONFIG.length; i++) {
-       if (FC.AUX_CONFIG[i] == 'ARM') {
-               if (bit_check(FC.CONFIG.mode, i))
-                       $(".armedicon").addClass('active');
-               else
-                       $(".armedicon").removeClass('active');
-       }
-       if (FC.AUX_CONFIG[i] == 'FAILSAFE') {
-               if (bit_check(FC.CONFIG.mode, i))
-                       $(".failsafeicon").addClass('active');
-               else
-                       $(".failsafeicon").removeClass('active');
-       }
+    for (const index in FC.AUX_CONFIG) {
```

Comment:
Really not sure if this change is going in the right direction - we are replacing something that is not elegant but correct with something that is elegant but not totally correct and can easily break.